pulp_container/app/registry_api.py

```diff
@@ -383,6 +384,13 @@ def create_dr(self, path, request):
 
         return distribution, repository
 
+    def handle_old_manifest_version(self, media_type, pk):
+        """
+        For non supported manifest versions, instead of redirecting the request, return earlier.
+        """
+        if media_type == MEDIA_TYPE.MANIFEST_V1:
+            raise ManifestNotFound(reference=pk)
+
```

Comment:
If a manifest's media type is `MEDIA_TYPE.MANIFEST_V1`, we should not raise `ManifestNotFound`. Instead, let's return it because we have got the same media type as requested by a user. Also, the `media_type` variable can contain comma separated values.